docs/themis/installation/installation-from-packages.md

```diff
@@ -242,7 +242,7 @@ which can be used to redistribute Themis to systems without MSYS2.
  3. Use NSIS installer GUI (steps 1–3 below)
     to install Themis along with all necessary dependencies.
 
-![](https://docs.cossacklabs.com/files/wiki/cossack-labs-themis-win-installer.png)
+![](/files/wiki/cossack-labs-themis-win-installer.png)
```

Comment:
This one might be fishy.

Reply:
why fishy?

Reply:
Well, that's the only link to `/files/wiki/something` that uses `https://docs.cossacklabs.com` prefix.

Now that I've thought about it, I noticed that the file *is* hosted on `docs.cossacklabs.com`, not in this repository.

Reply:
> I guess we should upload it here...

strongly agree